buildbot/check.sh

```diff
@@ -33,4 +33,4 @@ if [ -z "${TESTCASE}" ]; then
     exit 1
 fi
 
-make ${TESTCASE} VERBOSE=1 LIT_ARGS="-v -j `nproc`"
+make ${TESTCASE} VERBOSE=1 -j`nproc` LIT_ARGS="-v"
```

Comment:
in both cases (old and new) the -j parameter is redundant. if it is not specified all  available CPUs are used.
we can also remove VERBOSE=1 because it brings to much useless output

Reply:
This patch enables parallel **BUILD** of the tools used in LIT tests.